features/resources/spatial_DecompositionTable.csv

```diff
@@ -43,4 +43,8 @@ IfcSite,IfcRailway
 IfcSite,IfcRoad
 IfcSite,IfcExternalSpatialElement
 IfcSite,IfcSpace
-IfcSpace,IfcSpace
\ No newline at end of file
+IfcSpace,IfcSpace
```

Comment:
I think this should be a separate table. IfcStair is not a spatial element. Or the table can be renamed to simply decomposition table. But people are going to complain if we don't have consistent vocabulary on these kind of things.

Reply:
Moved it to a separate table for now